packages/babel-types/src/definitions/core.js

```diff
@@ -525,8 +534,8 @@ defineType("MemberExpression", {
 defineType("NewExpression", { inherits: "CallExpression" });
 
 defineType("Program", {
-  visitor: ["directives", "body"],
-  builder: ["body", "directives", "sourceType"],
+  visitor: ["interpreter", "directives", "body"],
```

Comment:
One comment here from Slack is we might not want to visit it, since it is comment-like and we don't visit other comment types. I don't feel strongly since it is trivial to reach from `program`